code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -39,7 +39,7 @@ true, and the mob is not yet deleted, so we need to check that as well*/
 
 		//blunt objects should really not be embedding in things unless a huge amount of force is involved
 
-		var/embed_threshold = weapon_sharp? 5*I.w_class : 15*I.w_class
```

Comment:
Seems like an odd change

Reply:
Why so?

Reply:
What are these numbers based on, if any at all?

Check what weapons this invalidates in being able to embed

Reply:
Wait, what do you think this is doing?

Reply:
from var/embed_chance = (damage - embed_threshold)*I.embed_mult 

If the embed threshold is greater than the damage, then it will never embed, so which weapons does this make it impossible to embed with now?

Reply:
It just makes more things harder to embed in general, proportionate to the item's size. 

Bigger things embed less, smaller things embed less. I could give you a list of every item that is impossible to embed but I'm not sure if it's worth the time other than to just say "Shit embeds less proportionately." Unless there is a tiny item that has massive damage, this just makes everything stick in at a flat %